vmdb/lib/tasks/evm_dba.rake

```diff
@@ -75,6 +75,10 @@ module EvmDba
   end
 end
 
+namespace :db do
+  task :migrate => "evm:db:environmentlegacykey"
```

Comment:
@kbrock Is this whacking the rails db:migrate rake task?

Maybe you want to try something like this instead to "enhance" it:

```ruby
# loads the v1 key into the enviroment
task :environmentlegacykey => :environment do
  MiqPassword.add_legacy_key('v0_key', :v0)
  MiqPassword.add_legacy_key('v1_key')
end

Rake::Task["db:migrate"].enhance ["evm:db:environmentlegacykey"]
```

See: http://edgar.tumblr.com/post/52300664342/how-to-extend-an-existing-rake-task